### PR TITLE
Maf88/172 add how to contribute file

### LINKED ISSUE #172: Add CONTRIBUTIONS file
title is self-explanatory

inspiration here: https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst

### PR DESCRIPTION
**Pull request by maf88**
Fix #172.

Based on spinal cord toolbox file
(https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst)

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -319,27 +319,28 @@ You **must** add Labels to PRs, as these are used to automatically generate Chan
 - **Cross-compatibility:** If your PR breaks cross-compatibility with a previous
   stable release of SCT, you should add the label ``compatibility``.
 
-`Here <https://github.com/neuropoly/axondeepseg/pull/44>`_ is an example of PR 
-with proper labels and description. (#TODO: Find a better example)
+`Here <https://github.com/neuropoly/axondeepseg/pull/44>`_ is an example of PR
+with proper labels and description. (#TODO: Find a better example)
```
Comment (maf88) [inline CONTRIBUTING.rst]: Is it a good example?

Comment (jcohenadad) [reply CONTRIBUTING.rst]: a bit old and has no label, use this instead:
https://github.com/neuropoly/axondeepseg/pull/176

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,346 @@
+=====================
+Contributing to SCT
```
Comment (jcohenadad) [inline CONTRIBUTING.rst]: replace by AxonDeepSeg

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -145,75 +145,75 @@ Examples:
...
+ or ADS Wiki.
 
-  If conflict, address them.
+ If conflict, address them.
 
 - Please add tests, especially with new code:
 
-  #TODO: UPDATE PARAGRAPH FOR ADS As of now, we have integration tests (that run in `sct_testing`),
-  and unit tests (in `/unit_testing/`).
+ #TODO: UPDATE PARAGRAPH FOR ADS As of now, we have integration tests (that run in `sct_testing`),
```
Comment (maf88) [inline CONTRIBUTING.rst]: For ADS, we can say:
« 
As of now, we have integration tests and unit tests in the ``./test/`` folder.

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+- Make sure the PR changes are not in conflict with the documentation,
+ either documentation files (`/README.md`, `/documentation/`), program help,
+ or ADS Wiki.
+
+ If conflict, address them.
+
+- Please add tests, especially with new code:
+
+ #TODO: UPDATE PARAGRAPH FOR ADS As of now, we have integration tests (that run in `sct_testing`),
```
Comment (jcohenadad) [inline CONTRIBUTING.rst]: sct_testing?

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+- **Category:** Choose **one** label that describes the
+ `category <https://github.com/neuropoly/axondeepseg/wiki/Rules-for-commits-and-issues-labelling-(git)#issue-category>`_
+ (white font over purple background).
+
+- **ADS Function:** Choose one or multiple labels corresponding to the ADS
+  functions that are mainly affected by the PR (black font over light purple
+  background).
+
+- **Cross-compatibility:** If your PR breaks cross-compatibility with a previous
+  stable release of SCT, you should add the label ``compatibility``.
```
Comment (jcohenadad) [inline CONTRIBUTING.rst]: SCT...

**Comment by maf88** [comment]
I left 2 TODO at line 179 and 323 and explained my concern.

**Pull request merged**